### PR TITLE
Sentence case all the things analytics

### LINKED ISSUE #6186: Update all labels in the analytics experience to use sentence case
Update all the labels in the analytics section to use "Sentence case" for consistency.

### PR DESCRIPTION
**Pull request by LevinMedia**
Fixes #6186

This PR updates all labels and strings in the analytics section of WooCommerce Admin to use sentence case. This is part of a broader effort accross WooCommerce / WordPress to unify around this editorial style. 

For reference: p9Jlb4-1ly-p2#comment-1999

### Detailed test instructions:

-   run this branch
-  Check each page of the analytics, and all available sub-menus to ensure all strings are now presented in sentence case

### CODE REVIEW

**Comment by adrianduffell** [comment]
@elizaan36 What are your thoughts on this?

**Comment by elizaan36** [comment]
Looks good to me @LevinMedia !

For some reason the `Month to Date` is still showing as title case in the dropdown and table headers:
image
Also `Last Month` snuck in there.

image
image

I found some inconsistencies in the Products area. The dropdown:
image
And the footer:
image
For the footer, the first word would be capitalized rather than all lower case.

**Comment by LevinMedia** [comment]
Thanks for picking those out @elizaan36 I've updated all those items in my last commit, and caught a few too.

**Comment by mattsherman** [comment]
Just a reminder that making these changes will require retranslation of all affected strings. I imagine we should make sure that this is not merged until the translation are available, though I will admit that I am not familiar with the translation process for WooCommerce, so perhaps this isn't an issue?

**Comment by timmyc** [comment]
Excellent point @mattsherman - in the past we haven't held up releases or features based upon string translation, we could try and see what % of total strings are modified here as that could cause some languages that are "border line" in % translated to fall behind. Then again when we first shipped wc-admin we didn't hold up anything due to translations so I don't think we need to be too concerned here.

File: docs/woocommerce.com/analytics-customers-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-customers-report.md:15]: Should the documentation headings stay capitalized?

Comment (LevinMedia) [reply docs/woocommerce.com/analytics-customers-report.md:15]: @adrianduffell I'd say since we're sentence casing all the things in product, it's okay to sentence case them in our docs too.

Comment (adrianduffell) [reply docs/woocommerce.com/analytics-customers-report.md:15]: I'm not sure. I feel like a documentation heading should remain title case since it's the more common style for this context.

This probably needs a second opinion 🙂 

cc @woocommerce/isotope 

If we change this instance to sentence case, for consistency I think all of the other documentation headings need to be changed too. There are still instances of title case in this file and elsewhere. For example:

Screen Shot 2021-03-30 at 10 09 11 am

Screen Shot 2021-03-30 at 10 09 15 am

Screen Shot 2021-03-30 at 10 09 25 am

Screen Shot 2021-03-30 at 10 09 29 am

Comment (moon0326) [reply docs/woocommerce.com/analytics-customers-report.md:15]: I also think the documentation heading should remain title case as all other docs in our repository use title case.

**Comment by LevinMedia** [comment]
@moon0326 @adrianduffell - just pushed a commit that updates the doc titles back to title case 👍 I think I got em all.

File: docs/woocommerce.com/analytics-downloads-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-downloads-report.md:9]:
```suggestion
![Downloads Report All Downloads View](images/analytics-downloads-report.png)
```

File: docs/woocommerce.com/analytics-downloads-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-downloads-report.md:20]:
```suggestion
![Downloads Report Advanced Filters](images/analytics-downloads-report-advanced-filters.png)
```

File: docs/woocommerce.com/analytics-orders-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-orders-report.md:29]:
```suggestion
![Customers Report Advanced Filters](images/analytics-orders-report-advanced-filters.png)
```

File: docs/woocommerce.com/analytics-products-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-products-report.md:7]:
```suggestion
By default, the Products Report displays the `All products` view. All products that have had sales in the specified date range will be shown.
```

File: docs/woocommerce.com/analytics-products-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-products-report.md:9]:
```suggestion
![Products Report All Products View](images/analytics-products-report.png)
```

File: docs/woocommerce.com/analytics-products-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-products-report.md:13]:
```suggestion
![Products Report Single Product Search](images/analytics-products-report-single-product-search.png)
```

File: docs/woocommerce.com/analytics-products-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-products-report.md:17]:
```suggestion
![Products Report Single Product View](images/analytics-products-report-single-product.png)
```

File: docs/woocommerce.com/analytics-taxes-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-taxes-report.md:5]:
```suggestion
### All Taxes View
```

File: docs/woocommerce.com/analytics-taxes-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-taxes-report.md:7]:
```suggestion
By default, the Taxes Report displays the `All taxes` view. All taxes that have been collected in the specified date range will be shown.
```

File: docs/woocommerce.com/analytics-taxes-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-taxes-report.md:9]:
```suggestion
![Taxes Report All Taxes View](images/analytics-taxes-report.png)
```

File: docs/woocommerce.com/analytics-variations-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-variations-report.md:7]:
```suggestion
By default, the Variations Report displays the `All variations` view. All variations that have had sales in the specified date range will be shown.
```

File: docs/woocommerce.com/analytics-variations-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-variations-report.md:9]:
```suggestion
![Variations Report All Variations View](images/analytics-variations-report.png)
```

File: docs/woocommerce.com/analytics-variations-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-variations-report.md:13]:
```suggestion
![Variations Report Single Variation Search](images/analytics-variations-report-single-variation-search.png)
```

File: docs/woocommerce.com/analytics-variations-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-variations-report.md:44]:
```suggestion
![Variations Report Advanced filters](images/analytics-variations-report-advanced-filters.png)
```

File: packages/components/src/advanced-filters/README.md
Comment (adrianduffell) [inline packages/components/src/advanced-filters/README.md:1]:
```suggestion
# Advanced Filters
```

File: packages/components/src/calendar/stories/date-range.js
Comment (adrianduffell) [inline packages/components/src/calendar/stories/date-range.js:25]:
```suggestion
			<H>Date range picker</H>
```

File: packages/components/src/table/README.md
Comment (adrianduffell) [inline packages/components/src/table/README.md:43]:
```suggestion
	title="Revenue last week"
```

File: packages/components/src/table/README.md
Comment (adrianduffell) [inline packages/components/src/table/README.md:134]:
```suggestion
	caption="Revenue last week"
```

File: packages/components/src/table/README.md
Comment (adrianduffell) [inline packages/components/src/table/README.md:213]:
```suggestion
	caption="Revenue last week"
```

File: packages/components/src/table/stories/table-card.js
Comment (adrianduffell) [inline packages/components/src/table/stories/table-card.js:22]:
```suggestion
		title="Revenue last week"
```

File: packages/components/src/table/stories/table-placeholder.js
Comment (adrianduffell) [inline packages/components/src/table/stories/table-placeholder.js:14]:
```suggestion
		<TablePlaceholder caption="Revenue last week" headers={ headers } />
```

File: packages/components/src/table/stories/table.js
Comment (adrianduffell) [inline packages/components/src/table/stories/table.js:14]:
```suggestion
		<Table caption="Revenue last week" rows={ rows } headers={ headers } />
```

File: readme.txt
Comment (adrianduffell) [inline readme.txt:170]: I don't think we should modify historical CHANGELOG entries.

**Review by adrianduffell — CHANGES_REQUESTED** [review]
@LevinMedia Thanks for updating the documentation back to title case. I think this is close!

I found some other things in the documentation and made code suggestions for them. The PR is also touching previous CHANGELOG entries, which probably isn't ideal?

**Comment by adrianduffell** [comment]
I think it would be worth committing my suggested changes before merging. It should be pretty quick [applying them as a batch in GitHub](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).

**Comment by LevinMedia** [comment]
@adrianduffell @samueljseay I'm with both of you there - I'll carve out time to put this one to bed this week. Thanks for your patience!

**Comment by samueljseay** [comment]
@LevinMedia I know you're busy but just thought I'd check up on this one. It would be sad not to get these changes in

**Comment by joshuatf** [comment]
This PR has been rebased and @adrianduffell's feedback has been addressed.  @adrianduffell @samueljseay would you mind give this a look and ✅  if things look okay?

File: docs/woocommerce.com/analytics-customers-report.md
Comment (adrianduffell) [inline docs/woocommerce.com/analytics-customers-report.md:29]:
```suggestion
![Customers Report Advanced Filters](images/analytics-customers-report-advanced-filters.png)
```

File: readme.txt
Comment (adrianduffell) [inline readme.txt:462]:
```suggestion
- Fix: Add Customer Type column to the Orders report table. #5820
```

File: readme.txt
Comment (adrianduffell) [inline readme.txt:515]:
```suggestion
- Fix: Add Customer Type column to the Orders report table. #5820
```

File: readme.txt
Comment (adrianduffell) [inline readme.txt:735]:
```suggestion
- Fix: Monetary Advanced Filters in Customers Report with correct currency object prop. #4356
```

File: readme.txt
Comment (adrianduffell) [inline readme.txt:1241]:
```suggestion
- Dev: Advanced Filters: Add Tracks events #2525 (Components, Packages)
```

File: readme.txt
Comment (adrianduffell) [inline readme.txt:1332]:
```suggestion
- Dev: Replace Rebuild reports settings with Import historical Data #2198
```

**Review by adrianduffell — COMMENTED** [review]
Awesome this is great thanks @joshuatf 🙌 

I see a few accidental changes still remaining in the changelog and ALT text for some images in .md files. I'll make the changes for these and then I think this is good to go!

**Review by adrianduffell — APPROVED** [review]
@joshuatf @LevinMedia this LGTM, thanks for your work on this! 🚀 

Pre-approving but could you check on the fixes I made to the changelog and ALT text in these commits before merging?

ed1544e40024f66ba0e906c0d282af2d37627ada 953e0c015cf34c32650a1a004bcb37c3e207acc9 72d104b384f9850f0200cad3284776b5f2b22c32

**Comment by joshuatf** [comment]
Thanks for pushing those reversions, @adrianduffell!  Looks good to me.

**Comment by LevinMedia** [comment]
Thanks for bringing this over the line @joshuatf @adrianduffell ❤️

**Pull request merged**
